### PR TITLE
Mk merchpage

### PR DESCRIPTION
**Pull request by Mehret17**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,8 +1,61 @@
 function writeToDom(domString, domId){
   var myDiv = document.getElementById(domId);
-  myDiv.innerHTML += domString;
+  myDiv.innerHTML = domString;
 }
```
Comment (michaelclark2) [inline main.js]: this writeToDom wont work if it doesnt check if the div is there first, only loads the last function called

Comment (Mehret17) [reply main.js]: ok - fixed it

**Comment by michaelclark2** [comment]
looks good :+1:

**Comment by michaelclark2** [comment]
👍 👍 👍

**Comment by Mehret17** [comment]
closes #5

**Pull request merged**